docs/reference/connectors/clickhouse.md

```diff
@@ -0,0 +1,117 @@
+---
+sidebar_label: elastic
+---
+
+# The `clickhouse` Connector
+
+The `clickhouse` collector aims integrate the [ClickHouse] database in Tremor. It has been tested with ClickHouse `v22.3-lts`.
+
+[Clickhouse]: https://clickhouse.com/
+
+This connector is a sink-only connector.
+
+## Configuration
+
+```troy
+  # File: config.troy
+  define connector clickhouse from clickhouse
+  with
+    config = {
+      # The hostname of the database node
+      "url": "localhost",
+
+      # The database to write data to
+      "database": "tremor_testing",
+
+      # The table to write data to
+      "table": "people",
+      "columns": [
+        # All the table's columns
+        {
+          # The column name
+          "name": "name",
+          # Its type
+          "type": "String",
+        }
+        {
+          "name": "age",
+          "type": "UInt8",
+        },
+      ]
+    }
+  end;
+```
+
+## Value conversion
+
+The following sections show how Tremor values are transformed into ClickHouse values. As numerous casts can be performed, the conversions are grouped by output type.
+
+Any type which is not documented in the following sections is considered as unsupported.
+
+### [`String`][CString]
+
+[CString]: https://clickhouse.com/docs/en/sql-reference/data-types/string
+
+ClickHouse `String`s values can be created from any Tremor string.
+
+### [`UInt64`, `UInt64`][CNumerals]
+
+[CNumerals]: https://clickhouse.com/docs/en/sql-reference/data-types/int-uint
+
+`Int64`s can be created from any Tremor integer which is not greater than 2^63 - 1 (9223372036854775807).
+
+`UInt64`s can be created from any non-negative Tremor number.
+
+
+### [`DateTime`][CDateTime]
+
+[CDateTime]: https://clickhouse.com/docs/en/sql-reference/data-types/datetime
+
+`DateTime`s can be created from any non-negative Tremor integer. It represents the number of seconds elapsed since January 1st of 1970 at 00:00:00, in UTC timezone.
```

Comment:
```suggestion
`DateTime`s can be created from any non-negative Tremor integer. It represents the number of seconds elapsed since January 1st of 1970 at 00:00:00, in UTC timezone. It is encoded as a 32 bit unsigned integer.
```